### PR TITLE
Fix evaluatedValuePopup floats away when scrolling

### LINKED ISSUE #2564: [Bug] Evaluated pane floats away when said property is scrolled outside of view (screenshot)
## Description

Evaluated pane floats away when a property is scrolled outside of view
Screenshot 2021-01-13 at 6 54 06 PM

### Steps to reproduce the behavior:
1. Open property pane for the table widget.
2. Input a few lines of text inside table data
3. Scroll while the evaluated value pane is visible.

### PR DESCRIPTION
**Pull request by SatishGandham**
## Description
**Use property pane as boundary for the evaluated popup**
Evaluated value popup floats away when scrolling, this PR sets the property pane as its boundary.

Fixes #2564

## Type of change

- Bug fix that could have impact in other areas.
 
## How Has This Been Tested?

- Checking the scroll behaviour before and after.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: fix/evaluated-value-popup-floats-away 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 51.58 **(0.01)** | 32.75 **(0.02)** | 29.66 **(0.01)** | 52.08 **(0.01)**
 :green_circle: | app/client/src/components/editorComponents/CodeEditor/EvaluatedValuePopup.tsx | 66.28 **(2.08)** | 59.42 **(2.5)** | 44.83 **(1.97)** | 68.75 **(2.08)**
 :green_circle: | app/client/src/components/editorComponents/CodeEditor/index.tsx | 75.46 **(0.11)** | 52.89 **(0)** | 58.06 **(0)** | 76.12 **(0.12)**
 :red_circle: | app/client/src/components/propertyControls/CodeEditorControl.tsx | 37.93 **(-1.36)** | 0 **(0)** | 50 **(0)** | 41.67 **(-1.81)**
 :green_circle: | app/client/src/components/propertyControls/InputTextControl.tsx | 78.38 **(0.6)** | 28.57 **(0)** | 66.67 **(0)** | 77.14 **(0.67)**
 :green_circle: | app/client/src/pages/Editor/PropertyPane/index.tsx | 83 **(0.17)** | 71.58 **(0)** | 68 **(0)** | 83.87 **(0.17)**</details>

### CODE REVIEW

File: app/client/src/components/editorComponents/CodeEditor/EvaluatedValuePopup.tsx [hunk 1]
```diff
@@ -330,7 +336,13 @@ function EvaluatedValuePopup(props: Props) {
           modifiers={{
             offset: {
               enabled: true,
-              offset: "0, 15",
+              offset: offset,
+              order: 120,
```
Comment (riodeuno) [inline app/client/src/components/editorComponents/CodeEditor/EvaluatedValuePopup.tsx:340]: Let's make these modifiers constant using `useMemo`

**Review by riodeuno — DISMISSED** [review]
One comment. Rest looks good. Thanks for fixing a long standing bug!
Approving for tests.

**Comment by vivekverma2312** [comment]
@SatishGandham The issue is fixed that evaluated pane is not floating now but a new issue is observed that some part of evaluated pane is overlapping the property pane. 

Screenshot 2021-06-01 at 7 21 12 PM

Screenshot 2021-06-01 at 7 37 40 PM

**Comment by SatishGandham** [comment]
@vivekverma2312 , Please test this in the API screen as well.

**Comment by vivekverma2312** [comment]
@SatishGandham Tested and working fine now.

**Review by riodeuno — APPROVED** [review]

**Review by riodeuno — APPROVED** [review]

**Comment by mohanarpit** [comment]
@SatishGandham Can you please merge the latest release into this branch & try to close this PR?

**Comment by SatishGandham** [comment]
This fix has issues in the API editor. Will come back to it later, closing for now.

**Pull request closed without merging**